src/transforms/glossary.ts

```diff
@@ -27,11 +27,33 @@ export function addGlossaryDefinition(
 
         if (at == null) return
 
+        const fragment: Descendant[] = Editor.fragment(editor, at)
+
+        let nodeContent: any
+        const getToBottom = (item: Descendant[]) => {
+            if (Array.isArray(item)) {
+                item.some((elem: any) => {
+                    if (elem.type === "paragraph"
+                    ||(elem.type === "caption"
+                    || elem.type === "media_alt"
+                    || elem.type === "title"
+                    || elem.type === "preformat"
+                    || elem.type === "table_entry"
+                    || elem.type === "definition_term")) {
+                        nodeContent = JSON.parse(JSON.stringify(elem.children))
+                        return true
+                    }
+                    return getToBottom(elem.children)
+                })
+            }
```

Comment:
It might be better to use an array of validators. Something like: `const VALID_DEFINITION_TERM_ELEMENTS = [Paragraph.isParagraph, Title.isTitle...]`.

Why do you check for `elem.type === "definition_term"`?

`nodeContent = elem.children` should be fine

Reply:
Why do you check for elem.type === "definition_term"? : in case someone wants to copy the definition - maybe someone will come up with such an idea :). Should I get rid of it?

nodeContent = elem.children does not work, just like [...elem.children], this object is not extensible

Array of validators is great idea, I will do it this way